modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/TypeScriptReduxQueryClientCodegen.java

```diff
@@ -0,0 +1,292 @@
+/*
+ * Copyright 2018 OpenAPI-Generator Contributors (https://openapi-generator.tech)
+ * Copyright 2018 SmartBear Software
```

Comment:
Please remove line 3 as per https://opensource.stackexchange.com/questions/7300/copyright-notice-in-the-file-header-apache-v2-license/7301#7301

Reply:
@wing328 Since this was copied from the `typescript-fetch` version as a starting point, does this still hold true?

Reply:
I don't know. I'm not an expert in software licensing.